src/domUpdates.js

```diff
@@ -1,8 +1,8 @@
-import scripts from './scripts';
-import Pantry from './pantry';
-import Recipe from './recipe';
-import User from './user';
-import Cookbook from './cookbook';
+// import scripts from './scripts';
```

Comment:
Now that we have time we may check in with educators to see if there is a work around for this?